src/MeshVolume.ts

```diff
@@ -119,7 +119,7 @@ export default class MeshVolume {
     // no op
   }
 
-  setAxisClip(axis: number, minval: number, maxval: number, _isOrthoAxis: boolean): void {
+  setAxisClip(axis: "x" | "y" | "z", minval: number, maxval: number, _isOrthoAxis: boolean): void {
```

Comment:
Did you make sure that the bounds.bmin and bounds.bmax are types that are keyed with those letters?

Reply:
Yep, `Bounds` is two `Vector3`s